data/hai/unfettered 0 prologue.txt

```diff
@@ -1879,26 +1880,123 @@ mission "Unfettered: Solifuge 2: Auxiliary Design"
+	landing
+	invisible
+	to offer
+		has "Unfettered: Solifuge 2: Auxiliary Design: done"
+	on enter "Almaaz" "Arneb" "Alninam"
+		dialog "As you enter the system it occurs to you the nearby pirates may have information regarding that Auxiliary."
+	on enter "Alcyone" "Misam"
+		dialog "Entering the system you recall that the pirates of Misam seemed to be in the business of knowing things."
+	to fail
+		has "unfettered: it's the Alphas!"
+
+mission "Unfettered: No Answers Available"
+	landing
+	invisible
+	to offer
+		has "Unfettered: Questions Best Left Unanswered: active"
+	source
+		government "Pirate"
+	on offer
+		conversation
+			branch Misam
+				has "flagship planet: New Tortuga"
+			`You ask questions around but nobody knows anything about an Auxiliary here, or at least they're not willing to tell you. You may have more luck at another planet.`
+				defer
+			label Misam
+			branch "okay boy!"
+				not "midas: wronged"
+			branch "dead, boy!"
+				"midas: wronged" > 1
+			`You ask around for this Auxiliary; 'Settler', but no-one will reveal anything about it to you, until, suspecting you will not leave until you have something, someone comes, their face conceiled. "You. You have betrayed us by revealing our plan to the governor, you are not to be trusted. That said, those that you are looking for are our enemies as well, so I will reveal you the location you need -if- you agree to give us compensation for your mistake. We demand 10 millions, in the form of a fine, to make sure you will have to pay us interests should you lack money."`
+			choice
+				`	"I will give you your 10 millions.`
+				`	"I refuse. I will find my own way to find who is in control of that ship, however hard that may be."`
+					decline
+			action
+				fine 10000000
+			`	Upon verification of the payment, you receive a set of coordinates on this world.`
+			label "okay boy!"
+			action
+				set "unfettered: it's the Alphas!"
+			`You head to the given coordinates on the planet, but you cannot see anyone there. You wait around for an hour or so until, looking around, you finally spot it: there is a graffiti on the wall, right at the given coordinate. At first you did not pay too much attention to it because this kind of thing is common on pirate worlds, but now you notice it depicts a green human. The message is clear.`
+				decline
+			label "dead, boy!"
+			`As soon as you set foot on the planet, a team of mercenaries comes out of the shadows, armored with the same kind of armor Si-Ra had. Their weapons may holstered but you can feel the tension in the air. Their leader speaks up. "We know your query. Consider yourself lucky to not be killed on the spot. We do not guarantee that will not happen, should you land again."`
+			`	They evaluate their surrendings before coming closer to you, and once they are a meter a way they stop, clearly speaking their words as a threat, with a pause between each of them "Nemo. Me. Impune. Laccesit."`
```

Comment:
```suggestion
			`	They evaluate their surroundings before coming closer to you, and once they are a meter away they stop, clearly speaking their words as a threat, with a pause between each of them "Nemo. Me. Impune. Laccesit."`
```
Why does it feel like you were half asleep writing this mission?

Reply:
~~I have no idea what you're speaking about~~